webview/src/experiments/components/table/styles.module.scss

```diff
@@ -997,4 +997,10 @@ $badge-size: 0.85rem;
 
 .addConfigButton {
   margin: 20px auto;
+  text-align: center;
```

Comment:
Should we center this info relative to the table instead of the webview?

<img width="1422" alt="Screenshot 2023-02-17 at 9 20 05 AM" src="https://user-images.githubusercontent.com/43496356/219694499-99d87836-c14e-423d-ad4d-af88e4a0cb51.png">

How it would look: 

<img width="1361" alt="Screenshot 2023-02-17 at 9 25 32 AM" src="https://user-images.githubusercontent.com/43496356/219695544-32e87457-8dbb-4983-8d5d-7531d90f1422.png">

Reply:
Made the table scrollable instead of the webview. It now looks like:


https://user-images.githubusercontent.com/3683420/219703490-fe93a4d2-4a9b-4618-8188-3fdfd9780662.mov

What do you think? I think we previously made the table 100% of the height to avoid this, but if there is other content in the page, this makes more sense.(cc @shcheklein)

Reply:
Hmmm, I'm still seeing the same result (I'm assuming because the table content is pretty small):

<img width="1358" alt="Screenshot 2023-02-17 at 10 25 14 AM" src="https://user-images.githubusercontent.com/43496356/219709031-c226d207-43f4-4a0f-b6b0-c3269ea38a61.png">

Though I now realize why it'd be difficult to make info relative to the table if the table took up more than the width of the webview 🤔 Probably be best (and easiest since this is minor and I'm guessing we don't want to spend too much time on it 😄) to just leave it centered relative to the webview or left align the info (not sure how that would look though).

Reply:
I think that I didn't take the time to understand your first concern, I saw the opposite of what you were showing (which was still a problem). I see it now. IMO it's better to center the "Add a Pipeline Stage" section with the webview, but this isn't a strong opinion. I feel like there might be something to do with the table. Center it? Continue the lines until the end of the webview? I don't know.

Reply:
Quick test for what it'd look like centered 

https://user-images.githubusercontent.com/3683420/219748144-059ab35d-816c-4ce4-9bb3-356351ee55a4.mov

Reply:
Quick test for full width rows (there are probably other  borders not extending correctly like the previous commits one, this is a prototype). This might be the cleanest solution.


https://user-images.githubusercontent.com/3683420/219755714-6102c28a-440a-4f28-9f55-7ca5e30cd0f0.mov

Reply:
I went with that last option after all. Don't mind changing it in a follow-up if needed.

https://user-images.githubusercontent.com/3683420/219757613-3fa10ac5-e305-4a76-af71-e61538100de3.mov

Reply:
![image](https://user-images.githubusercontent.com/37993418/219982115-ec888447-c300-495a-ba7c-5c1e8f04a6d0.png)

I feel like this looks like a bug. We might be optimising for very artificial situations. However, there are two things that we do need to keep in mind: 

1. Users should not be in this state for very long.
2. Not everyone will have the sidebar open and some have wide screens. E.g

![Screenshot 2023-02-20 at 10 39 12 am (2)](https://user-images.githubusercontent.com/37993418/219982423-ff3cb794-cc26-4206-a7c3-b9ed96273648.png)

I don't think we should optimise the overall experience for when the button is shown. IMO the original solution was fine.

Reply:
I don't know, I think completing the rows look nice and feel like all the space is used, even on wide screens. I'll revert and keep the original solution, we can change it back later.

Reply:
Agreeing with Matt! This solution causes multiple styling bugs when it comes to the header cells. Plus we have to make the table scrollable when its easier to navigate when the webview is scrollable. 

I vote keep original solution! Personally, I think left aligning the text looks fine if we want to consider that in the future as well. 

<img width="795" alt="Screenshot 2023-02-20 at 8 20 03 AM" src="https://user-images.githubusercontent.com/43496356/220132176-ae8a6fba-dc20-4eb4-b2e8-38db4acfb98b.png">

Reply:
It's reverted, but I'm not sure I understand the styling and scrolling bugs. I'm seeing only one that would have been really easy to fix (rows to go pass 100% of width). All I did was make the table take the full width of the webview (minimum). IMO it makes the table feel less broken if it's small. All of our webviews utilize the full width.

Aligning left for button and text like that isn't found anywhere else in the extension, it's always center-aligned. It feels like we forgot to add the styles or the styles are broken if we align it that way.